src/components/EditorPage/EditorPage.tsx

```diff
@@ -15,14 +15,11 @@ export default class EditorPage extends React.PureComponent<Props> {
   componentWillMount() {
     this.props.onLoadAssetPacks()
     this.props.onBindKeyboardShortcuts()
-    document.body.classList.add('lock-scroll')
-    document.body.scrollTop = 0
```

Comment:
I'm not sure if this was preventive or a fix for a particular case. It might still be useful cc: @cazala

Reply:
Yep that one came in the search PR and was partially removed in the latest fix PR. Now that scrolling should be fixed we can validate that this solution works, previously it jumped around and left the page scroll locked incorrectly (probably due to the other overflow issues).

Reply:
Yes, the purpose of the `.lock-scroll` class  was to disable the "elastic scroll" effect on macOS while zooming in and out in the editor (you can see it happening in the second gif on this PR)